### DISCUSSION TITLE
CarPlay support coming?

### DISCUSSION THREAD

**Discussion by argash** [advplyr/audiobookshelf-app#81] [Ideas]
Just got the test flight this morning, thank you! First thing I noticed off the bat is that CarPlay is not yet supported. Is this a planned feature?

**Comment by advplyr** [comment]
I'm glad it is working. Yes, CarPlay will be supported eventually. I have to say Android Auto was not fun to build.. so hopefully Apple has a better experience with CarPlay.

**Reply by advplyr** [reply]
I couldn't get the media controls working that show on the lock screen and control center. Can you confirm that this is also not working for you?

**Reply by argash** [reply]
Good question I will test after I finish my current book some time tonight or tomorrow.

**Reply by argash** [reply]
ok, yup, nothing on lock screen or control center.

**Reply by lseufer** [reply]
Hi guys, any update on CarPlay?

**Reply by advplyr** [reply]
No updates, development for CarPlay hasn't started yet.

**Reply by adaramola** [reply]
Hello, I've been using your app since June or July and its great. I was wondering if there has been any movement on the CarPlay support? If not, do you think it will get some attention at some point? I ask with the understanding that I I'm pretty sure you're a one man shop and this is probably a side project for you. My question shouldn't be considered pressure, just me trying to get a better understanding of what to expect.

**Reply by Hafumoto** [reply]
CarPlay is currently the only thing holding me back, forcing me to use Plex and Prologue. I use this app whenever I'm home or moving by foot, but CarPlay is essential when driving. Having to use two separate server and apps is a bit of a hassle.

**Comment by Justanotherhomosapien** [comment]
Would happily donate towards a CarPlay feature.

**Reply by GOvEy1nw** [reply]
Yes same. It's an essential feature personally.

**Comment by stusemple** [comment]
If you wanted to set up a donation for car play I would also contribute. Love the app but still need prologue for car play.

**Reply by ghost** [reply]
Same here. It's a must have feature for me.

**Comment by Losicek** [comment]
HI :-) 
Just want to ask how it looks like with Carplay? :-) It would be awesome have it in car screen and not check phone. Thank you you are best!
